internal/pkg/fileutil/fileutil.go

```diff
@@ -0,0 +1,42 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package fileutil
+
+import (
+	"os"
+	"time"
+)
+
+// FileExists returns true if file/dir exists
+func FileExists(fp string) (ok bool, err error) {
+	if _, err := os.Stat(fp); err == nil {
+		ok = true
+	} else if os.IsNotExist(err) {
+		err = nil
+	}
+	return ok, err
+}
+
+// GetModTime returns file modification time
+func GetModTime(fp string) (time.Time, error) {
+	fi, err := os.Stat(fp)
+	if err != nil {
+		return time.Time{}, err
+	}
+	return fi.ModTime(), err
```

Comment:
```suggestion
	return fi.ModTime(), nil
```
(nitpick) i prefer explicit returns